ToDo.js

```diff
@@ -0,0 +1,376 @@
+            method: "GET"
+        });
+        const columnsAll = await response.json()
+        return columnsAll
+    }
+
+    async getCardsObj(colId) {
+        const response = await fetch(this.cardsPath + colId, {
+            method: "GET"
+        });
+        const cardsAll = await response.json()
+        return cardsAll
+    }
+
+    async sendColumnName(name, uuid) {
+        await fetch(this.columnsPath + uuid, {
+            method: "PUT",
+            body: name
+        })
+    }
+
+    async deleteColumn(columnId) {
+        await fetch(this.columnsPath + columnId, {
+            method: "DELETE",
+        })
+    }
+
+    async postNewColumn() {
+        const columnData = new FormData()
+        columnData.append("name", "Column")
+        await fetch(this.columnsPath, {
+            method: "POST",
+            body: columnData
+        })
+    }
+
+    async postCard(parentId, cardData) {
+        await fetch(this.cardsPath + parentId, {
+            method: "POST",
+            body: cardData
+        })
+    }
+
+    async putCard(modifiedCardData, cardId) {
+        await fetch(this.cardsPath + cardId, {
+            method: "PUT",
+            body: modifiedCardData
+        })
+    }
+
+    async deleteCard(cardId) {
+        await fetch(this.cardsPath + cardId, {
+            method: "DELETE",
+        })
+    }
+
+}
+
+class Modal {
+    modalContainer;
+    modalHTML;
+
+    constructor() {
+        this.modalContainer = document.querySelector(".modal-container")
+        this.modalHTML = ` <div class="modal fade" id="cardModal" tabindex="-1" aria-labelledby="cardModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+          <div class="modal-content">
+            <form class="card-form">
+              <div class="form-group py-3">
+                <label for="taskTitle">Title</label>
+                <input type="text" class="form-control card-title" id="taskTitle" placeholder="Task title">
+              </div>
+              <div class="form-group">
+                <label for="taskDescription">Description</label>
+                <input type="text" class="form-control card-description" id="taskDescription"
+                  placeholder="Task description">
+              </div>
+            </form>
+            <div class="card-modal__footer d-flex justify-content-end">
+              <button type="button" class="btn modal__btn-delete position-absolute" data-bs-dismiss="modal">delete</button>
+              <button type="button" class="btn modal__btn-cancel card-modal__btn" data-bs-dismiss="modal">cancel</button>
+              <button type="button" class="btn modal__btn-save card-modal__btn" data-bs-dismiss="modal">save</button>
+            </div>
+          </div>
+        </div>
+       </div>`
+    }
+
+    generateModal() {
+        this.modalContainer.innerHTML = ""
+        this.modalContainer.insertAdjacentHTML('beforeend', `${this.modalHTML}`)
+    }
+}
```

Comment:
This modal implementation is very concrete, you'd normally want to abstract it's functionalities for further reuse and, eventually create desired modal.